Signal/src/AppDelegate.m

```diff
@@ -317,7 +317,13 @@ - (void)application:(UIApplication *)application
  */
 - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray * _Nullable))restorationHandler
 {
-    DDLogWarn(@"%@ called %s with userActivity: %@, but not yet supported.", self.tag, __PRETTY_FUNCTION__, userActivity);
+    if ([userActivity.activityType isEqualToString:@"INStartVideoCallIntent"]) {
+        [[Environment getCurrent].callService handleCallKitStartVideo];
```

Comment:
AFAIK this is the only way to observe when user taps video button in CallKit ui.